src/solvers/dg_unstructured_quad/dg_2d.jl

```diff
@@ -73,18 +76,69 @@ function rhs!(du, u, t,
 end
 
 
-# flux differencing volume integral on curvilinear quadrilateral elements. Averaging of the
-# mapping terms, stored in `contravariant_vectors`, is peeled apart from the evaluation of
-# the physical fluxes in each Cartesian direction
-function calc_volume_integral!(du, u,
-                               mesh::Union{CurvedMesh{2}, UnstructuredQuadMesh},
-                               nonconservative_terms, equations,
-                               volume_integral::VolumeIntegralFluxDifferencing,
-                               dg::DGSEM, cache)
-  @threaded for element in eachelement(dg, cache)
-    split_form_kernel!(du, u, nonconservative_terms, volume_integral.volume_flux, element,
-                       mesh, equations, dg, cache)
+# Calculate 2D twopoint contravariant flux (element version)
+@inline function calcflux_twopoint!(ftilde1, ftilde2, u::AbstractArray{<:Any,4}, element,
+                                    volume_flux, mesh::Union{CurvedMesh{2}, UnstructuredQuadMesh},
+                                    equations, dg::DG, cache)
```

Comment:
```suggestion
                                    equations, dg::DGSEM, cache)
```
FDSBP methods will need to make of of the different sparsity structure to be efficient